### ISSUE TITLE
VAT api integration

### ISSUE THREAD

**Issue by vbezruchkin** [intelliants/subrion#134]
Probably this would be a good addition to the payment system we have in the core:
http://vatapi.com/

Implement as a plugin if there is any way.

**Comment by georgikolev7** [comment]
Good idea, I'm suggesting https://vatlayer.com/, it includes 100 free requests/monthly.
